packages/client/src/products/__tests__/getProductVariantsMeasurements.test.ts

```diff
@@ -5,40 +5,28 @@ import {
 } from 'tests/__fixtures__/products';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getProductVariantsMeasurements.fixtures';
-import moxios from 'moxios';
+import mswServer from '../../../tests/mswServer';
 
 describe('getProductVariantsMeasurements', () => {
   const expectedConfig = undefined;
   const spy = jest.spyOn(client, 'get');
 
-  beforeEach(() => {
-    moxios.install(client);
-    jest.clearAllMocks();
-  });
-
-  afterEach(() => moxios.uninstall(client));
+  beforeEach(jest.clearAllMocks);
 
   it('should handle a client request successfully', async () => {
-    const response = mockProductVariantsMeasurements;
-
-    fixtures.success({
-      id: mockProductId,
-      response,
-    });
+    mswServer.use(fixtures.success(mockProductVariantsMeasurements));
 
```

Comment:
This would look great with `expect.assertions(2);` here :)

Reply:
Added 🙂